tests/acceptance/features/apiSharingNg1/sharedByMe.feature

```diff
@@ -3863,3 +3863,806 @@ Feature: resources shared by user
+                "pattern": "^%file_id_pattern%$"
+              }
+            }
+          },
+          "permissions": {
+            "type": "array",
+            "minItems": 1,
+            "maxItems": 1,
+            "items": {
+              "type": "object",
+              "required": [
+                "@libre.graph.permissions.actions",
+                "grantedToV2",
+                "id",
+                "invitation"
+              ],
+              "properties": {
+                "@libre.graph.permissions.actions": {
+                  "type": "array",
+                  "enum": [
+                    [
+                      "libre.graph/driveItem/path/read",
+                      "libre.graph/driveItem/children/read",
+                      "libre.graph/driveItem/basic/read"
+                    ]
+                  ]
+                },
+                "grantedToV2": {
+                  "type": "object",
+                  "required": ["user"],
+                  "properties": {
+                    "user": {
+                      "type": "object",
+                      "required": ["displayName", "id"],
+                      "properties": {
+                        "id": {
+                          "type": "string",
+                          "pattern": "^%user_id_pattern%$"
+                        },
+                        "displayName": {
+                          "const": "Brian Murphy"
+                        }
+                      }
+                    }
+                  }
+                },
+                "id": {
+                  "type": "string",
+                  "pattern": "^%permissions_id_pattern%$"
+                },
+                "invitation": {
+                  "type": "object",
+                  "required": ["invitedBy"],
+                  "properties": {
+                    "invitedBy": {
+                      "type": "object",
+                      "required": ["user"],
+                      "properties": {
+                        "user": {
+                          "type": "object",
+                          "required": ["displayName", "id"],
+                          "properties": {
+                            "displayName": {
+                              "const": "Alice Hansen"
+                            },
+                            "id": {
+                              "type": "string",
+                              "pattern": "^%user_id_pattern%$"
+                            }
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "name": {
+            "const": "textfile.txt"
+          },
+          "size": {
+            "const": 12
+          }
+        }
+      }
+      """
+
+  @env-config
+  Scenario: sharee lists folder share with role Secure Viewer after share role is disabled (Project Space)
```

Comment:
we can do checks by both sharer and sharee in a single scenario for these kind of test scenarios.

Reply:
It should be sharer here. There's another PR https://github.com/owncloud/ocis/pull/10744 for sharee to check for shares `sharedWithMe`. Will update this on that PR.